packages/experimental/Menu/src/MenuList/MenuList.tsx

```diff
@@ -1,18 +1,45 @@
 /** @jsx withSlots */
 import React from 'react';
 import { View } from 'react-native';
-import { compose, UseSlots, withSlots } from '@fluentui-react-native/framework';
+import { compose, mergeProps, stagedComponent, UseSlots, withSlots } from '@fluentui-react-native/framework';
 import { menuListName, MenuListProps, MenuListType } from './MenuList.types';
 import { stylingSettings } from './MenuList.styling';
 import { MenuListProvider } from '../context/menuListContext';
 import { useMenuList } from './useMenuList';
-import { useMenuListContextValue } from './useMenuContextValue';
+import { useMenuListContextValue } from './useMenuListContextValue';
+import { IViewProps } from '@fluentui-react-native/adapters';
+
+const MenuStack = stagedComponent((props: React.PropsWithRef<IViewProps> & { gap?: number }) => {
+  const { gap, ...rest } = props;
+  return (final: React.PropsWithRef<IViewProps> & { gap?: number }, children: React.ReactNode) => {
+    if (gap && gap > 0 && children) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore - TODO, fix typing error
```

Comment:
Actual TODO or fine to keep?

Reply:
I copied this from Stack logic. I'll log an issue and replace the TODO.